Article/Article.js

```diff
@@ -26,4 +33,6 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document
+  .querySelectorAll(".article")
+  .forEach(a => new Article(a));
```

Comment:
You don't need to save the articles variable.  You never use it anywhere and .forEach actually doesn't return anything so if you console.log here, it'll say undefined.